services/utils.js

```diff
@@ -42,13 +42,17 @@ const objIsEdge = (obj, vertexIndex) => {
     if (!validFields.includes(fields[i])) return false;
   }
   const hasDestinationKey = fields.includes(GRAPH_FIELDS.DESTINATION);
-  const hasDestinationValue = obj[DESTINATION] !== null;
-  const isConnectedToSelf = obj[DESTINATION] === `${vertexIndex}`;
+  const hasDestinationValue = obj[GRAPH_FIELDS.DESTINATION] !== null;
+  const isConnectedToSelf = obj[GRAPH_FIELDS.DESTINATION] === `${vertexIndex}`;
 
   if (!hasDestinationKey || !hasDestinationValue || isConnectedToSelf)
     return false;
 
-  if (!fields.includes(WEIGHT) || isNaN(parseFloat(obj[WEIGHT]))) return false;
+  if (
+    !fields.includes(GRAPH_FIELDS.WEIGHT) ||
+    isNaN(parseFloat(obj[GRAPH_FIELDS.WEIGHT]))
+  )
+    return false;
```

Comment:
In here cant you just return the condition? (if so you should also save your code into variables to put into the if just like the pr before that)

Reply:
I took another look at the function, and realized it could be made simpler, if somewhat verbose. Is the new version better? Will upload very soon.